kafkazk/zookeeper.go

```diff
@@ -330,6 +247,41 @@ func (z *ZKHandler) GetReassignments() Reassignments {
 	return reassigns
 }
 
+// ListReassignments looks up any ongoing topic reassignments and returns the data
+// as a Reassignments. ListReassignments is a KIP-455 compatible call for Kafka
+// 2.4 and Kafka cli tools 2.6.
+func (z *ZKHandler) ListReassignments() (Reassignments, error) {
+	reassignments := Reassignments{}
+
+	var path = "/brokers/topics"
```

Comment:
path is unused

Reply:
Heh, good catch - it's dead code from when this was being tested I think with a single-topic reassignment lookup.